spec/app/ics-020-fungible-token-transfer/README.md

```diff
@@ -256,27 +320,58 @@ function sendFungibleTokens(
 
 ```typescript
 function onRecvPacket(packet: Packet) {
-  FungibleTokenPacketData data = packet.data
+  channel = provableStore.get(channelPath(portIdentifier, channelIdentifier))
+  if channel.version === "ics20-1" {
+      FungibleTokenPacketData data = packet.data
```

Comment:
Just to be explicit that the packet data is json marshalled and needs to be unanarshaled.

```suggestion
      FungibleTokenPacketData data = UnmarshalJSON(packet.data)
```